posts/inside-rust/2023-01-24-content-delivery-networks.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Diversifying our Content Delivery Networks
+author: Jan David Nose
+team: The Rust Infrastructure Team <https://www.rust-lang.org/governance/teams/infra>
+---
+
+Over the past few weeks, the [Infrastructure Team] has been working on a setting
+up a second [Content Delivery Network] for releases and crates.
```

Comment:
We should put "(CDN)" after the full name so that folks understand that CDN is used to refer to Content Delivery Network.

Reply:
Good one! :+1: